util/libcrypto.num

```diff
@@ -5536,4 +5536,6 @@ X509_STORE_CTX_set_get_crl              5663	3_2_0	EXIST::FUNCTION:
 X509_STORE_CTX_set_current_reasons      5664	3_2_0	EXIST::FUNCTION:
 OSSL_STORE_delete                       5665	3_2_0	EXIST::FUNCTION:
 BIO_ADDR_copy                           5666	3_2_0	EXIST::FUNCTION:SOCK
-EVP_DigestSqueeze                       ?	3_3_0	EXIST::FUNCTION:
+EVP_DigestSqueeze                       5667	3_3_0	EXIST::FUNCTION:
+EVP_aes_128_cbc_hmac_sha512             5668	3_3_0	EXIST::FUNCTION:
+EVP_aes_256_cbc_hmac_sha512             5669	3_3_0	EXIST::FUNCTION:
```

Comment:
The values should not be set. I.e., there should be question marks. If you delete all these three 3_3_0 entries and run `make update` does it add them with question marks?

Reply:
I didn't manually set these values, I used `make ordinals` to update libcrypto.num. I'm not sure why the 3_3_0 was put there. I used `make update` or `make ordinals` to update libcrypto.num just now, and it indeed added them with question marks.   I will update these items, thanks.